src/ocaml-config/ocaml_config.ml

```diff
@@ -575,6 +582,7 @@ let make vars =
       get_prog_or_dummy_exn vars "bytecomp_c_compiler"
     in
     let native_c_compiler = get_prog_or_dummy_exn vars "native_c_compiler" in
+    let native_pack_linker = get_prog_or_dummy_exn vars "native_pack_linker" in
```

Comment:
Is this field always set? If not, what happens if it's not?

Reply:
Looking at the history of [ocaml/utils/config.mlp](https://github.com/ocaml/ocaml/blob/trunk/utils/config.mlp), it looks like this field has existed for 14 years (exposed in this commit: https://github.com/ocaml/ocaml/commit/32abe25b89d7c58272a33a80519dbba9528f29f4). 
Because the value is used when running `ocamlopt -pack` I believe it is always set.

Reply:
14 y/o is good enough for me.